assets/course-theme/blocks/course-navigation.js

```diff
@@ -79,17 +79,17 @@ const Module = ( { title, lessons } ) => (
 	<section className="sensei-lms-course-navigation-module sensei-collapsible">
 		<header className="sensei-lms-course-navigation-module__header">
 			<div className="sensei-collapsible__toggle">
-				<h2 className="sensei-lms-course-navigation-module__title">
+				<div className="sensei-lms-course-navigation-module__title">
```

Comment:
It also changes this HTML in the JS side, so we keep server and front sync.

Reply:
Thanks, I missed that in the other PR.